src/loghandler.cpp

```diff
@@ -261,12 +261,20 @@ void LogHandler::addLog(const Log& log,
   emit logEntryAdded(buffer);
 
 #if defined(MZ_ANDROID)
+#  ifdef MZ_DEBUG
```

Comment:
Did you mean to commit this? This would log like Staging if we're running a release debug build, right?